opencog/cython/PythonEval.cc

```diff
@@ -415,8 +414,7 @@ Handle PythonEval::apply(const std::string& func, Handle varargs)
     // Py_DECREF(pyDict);
 
     // If we can't find that function then throw an exception.
-    if (!pyUserFunc)
-    {
+    if (!pyUserFunc) {
```

Comment:
I've been doing this all along you probably didn't notice because I was making other changes. I find the real cognitive load to be having things mix around so your eye has to look for one and the other at the same time.

I actually like the non-attached brace style that was there better, but I've been trying to conform to the standards which specify attaching braces when editing routines that are using different styles in one routine so at least that routine is consistent. So if you want that to be different, let's change the style on the wiki to be the better one. I had assumed you liked the other style for brevity, or something. It doesn't make sense to have coding standards that you don't like.

I'm happy to revert the above changes but I'd like to make the file be consistent in its use of one or the other. The only reason I've been nagging a bit about coding styles is because I'd like to write my code consistently and I just don't know what that style should be. I'm happy for that to be 100% Linas style. :-) As we seem to agree on more than I even thought.